tests/etl/test_transformers.py

```
@@ -0,0 +1,78 @@
+            column(
+                "c",
+                elements=st.one_of(
+                    st.datetimes(timezones=st.just(None)), st.text(), st.none()
+                ),
+            ),
+            column("d", elements=st.one_of(st.text(), st.none())),
+        ],
+        index=range_indexes(min_size=1, max_size=5),
+    )
+)
+def test_clean_and_convert_data_types(df: pd.DataFrame):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_clean_and_convert_data_types`

Add return type annotation: `None`

(ANN201)